plugins/notifications/src/components/NotificationsTable/NotificationsTable.tsx

```diff
@@ -237,6 +242,12 @@ export const NotificationsTable = ({
                     </Typography>
                   ) : null}
                   <Typography variant="caption">
+                    {!notification.user && (
+                      <>
+                        <BroadcastIcon className={classes.broadcastIcon} />
+                        &nbsp;
```

Comment:
Maybe consider a marginRight on the style instead

Reply:
done